guides/source/getting_started.md

```diff
@@ -1292,6 +1297,13 @@ link to make a `DELETE` request instead of a `GET` request.
 when the link is clicked. If the user cancels the dialog, the request will be
 aborted.
 
+To enable Turbo in this situation, we need to execute some install scripts.
+
+```bash
+rails importmap:install
+rails turbo:install stimulus:install
```

Comment:
I'm not sure about the need of installing `stimulus` here. I'm a newbie in Ruby & Rails ^^.

Reply:
After some tests, stimulus is not needed here.
```suggestion
rails turbo:install
```

But turbo:install warn about redis server not installed. Should we also add `turbo:install:redis` command here ?
```suggestion
rails turbo:install
rails turbo:install:redis
```

Reply:
~Can you post the output you get from `rails new` that requires you to later run these commands? As far as I know these are part of running `rails new` by default and if they aren't then that's a bug.~

Oh, this is a symptom of using `--skip-bundle`... that makes me even more hesitant to want to recommend that, since it ends up adding even more steps.